test/testdata/OWNERS

```diff
@@ -5,7 +5,6 @@ reviewers:
   - enj
   - liggitt
   - soltysh
-  - jim-minter
```

Comment:
same here (we really need to refactor our testdata dir into subdirs).  as it stands today, you should probably not remove yourself from this because devex owns at least half the files in the root testdata dir.